frontend/src/utils/institute/SldVerifyTabs.js

```diff
@@ -0,0 +1,15 @@
+
+/**
+ * stepStatus values: 'in-progress', 'not-started', 'complete'
+ */
+export const SLD_VERIFY_TABS = Object.freeze(
+  [
+    'FTE',
+    'French Programs',
+    'Career Programs',
+    'Indigenous Students & Support Programs',
+    'Special Education',
+    'English Language Learning',
+    'Refugee'
+  ]
+);
```

Comment:
oh this should be SDC_VERIFY_TABS (student-data-collection). 

There's a ticket to rename SLD -> SDC that incoming. 

I also think this can go into utils/sdc since it's related to collections? <-- new folder from the latest PR.